sky/jobs/utils.py

```diff
@@ -965,7 +965,7 @@ def get_hash(task):
         'STATUS',
     ]
     if show_all:
-        columns += ['STARTED', 'CLUSTER', 'REGION', 'DESCRIPTION']
+        columns += ['STARTED', 'CLUSTER', 'REGION', 'SCHED. STATE', 'DETAILS']
```

Comment:
minor: can be a future thing, we can even hide the `SCHED. STATE` behind another flag `sky jobs queue --debug`